### DISCUSSION TITLE
Number of Items, and Multiple Expirationdates for one Poduct

### DISCUSSION THREAD

**Discussion by cocker-cc** [lorenzovngl/FoodExpirationDates#88] [Ideas]
Of one particular Product I have Items with different Expirationdate.  

My Idea:  
- I can store multiple Expirationdates per Product
- together with the particular Expirationdate I can store the according Number of Items
- the Position in the List reflects the oldest Expirationdate, if the Number of Items of this Expirationdate falls down to `0` then this particular Expirationdate is removed, and the Product's Position in the List changes to the now oldest Expirationdate
- the List should show the Product and the Sum of all Items, and only when clicked on the Product, I can see how many Items for each particular Expirationdate are there

Example:
```
Chocolate-Cream: 2024-02-17: 2
Olive Oil: 2025-06-28: 2
Kidneybeans: 2026-05-20: 5
```

When I click on `Kidneybeans` it shows:
```
Kidneybeans:
2026-05-20: 2
2027-01-13: 3
```

Perfect would be Buttons with `+` and `-` to in- and decrease the Numbers.

**Comment by lorenzovngl** [comment]
Hello!
Thank you for this suggestion, I will consider it in the future.
